<launch>
    <node pkg="vins" type="vins_node" args="~/catkin_ws/src/VINS-Fusion/config/xtdrone_sitl/px4_sitl_mono_imu_config.yaml"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="base_link_to_camera_tf" args="0 0 0 0.5 -0.5 0.5 0.5 camera_link camera_depth" />
    <node pkg="movinav" type="monodepth.py" name="monodepth" output="screen" respawn="true">
        <param name="topic_color" value="/iris_0/stereo_camera/left/image_raw"/>
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find movinav)/config.rviz"/>
</launch>